Subject: [PATCH] feat: implement password setting (WIP)
---
Index: app/src/main/java/com/n1/moguchi/data/models/Parent.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/n1/moguchi/data/models/Parent.kt b/app/src/main/java/com/n1/moguchi/data/models/Parent.kt
--- a/app/src/main/java/com/n1/moguchi/data/models/Parent.kt	(revision b218d3ceb05cf52f805aff2bc0b51ac37cadf741)
+++ b/app/src/main/java/com/n1/moguchi/data/models/Parent.kt	(date 1701108612951)
@@ -5,5 +5,6 @@
     val userName: String = "",
     val email: String,
     val childrenList: List<Child>? = null,
-    val isParentMode: Boolean = true
+    val isParentMode: Boolean = true,
+    val parentModePassword: String? = null
 )
\ No newline at end of file
